src/Microsoft.Diagnostics.Runtime/src/DacInterface/SosDac.cs

```diff
@@ -726,7 +726,7 @@ public ulong GetMethodDescFromToken(ulong module, uint token)
         private delegate int DacGetIntPtr(IntPtr self, out IntPtr data);
 
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        private delegate int DacGetUlongWithArg(IntPtr self, ulong arg, out ulong data);
+        private delegate int DacGetUlongWithArg(IntPtr self, ulong arg, out ClrDataAddress data);
```

Comment:
Are you fine with the delegate name(s) which now seems to be a bit misleading?

Also, I wonder if it is your intent to leave args passed to the Dac as `ulong`. (I see no problem in practice.)

Reply:
The args we pass **to** the dac shouldn't matter.  On the dac side these are going to get casted down to 32bit integers/pointers so we shouldn't need to sign extend anything ourselves.  Please let me know if you find that's not the case because it would be very surprising...